includes/class-amp-theme-support.php

```diff
@@ -721,6 +838,27 @@ protected static function ensure_required_markup( DOMDocument $dom ) {
 		}
 	}
 
+	/**
+	 * Dequeue Customizer assets which are not necessary outside the preview iframe.
+	 *
+	 * Prevent enqueueing customize-preview styles if not in customizer preview iframe.
+	 * These are only needed for when there is live editing of content, such as selective refresh.
+	 *
+	 * @since 0.7
+	 */
+	public static function dequeue_customize_preview_scripts() {
+
+		// Dequeue styles unnecessary unless in customizer preview iframe when editing (such as for edit shortcuts).
+		if ( ! self::is_customize_preview_iframe() ) {
+			wp_dequeue_style( 'customize-preview' );
```

Comment:
What about just doing `wp_deregister_style( 'customize-preview' );` so that we don't have to loop through dependencies?

Reply:
That could work, but it would also result in something like Query Monitor issuing a warning about a dependency missing.